user-api/overlays/stage/role-binding.yaml

```diff
@@ -15,9 +15,23 @@ subjects:
 ---
 apiVersion: rbac.authorization.k8s.io/v1beta1
 kind: RoleBinding
+metadata:
+  name: user-api-pods-info
+  namespace: thoth-backend-stage
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: user-api-pods-info
+subjects:
+  - kind: ServiceAccount
+    name: user-api
+    namespace: thoth-frontend-stage
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: RoleBinding
 metadata:
   name: user-api-argo
-  namespace: thoth-frontend-stage
+  namespace: thoth-middletier-stage
```

Comment:
middletier?

Reply:
this was a missed in the manifest earlier
user-api doesn't create any workflow in frontend, this is for middletier , as package-extract is scheduled there.

Reply:
from now on user-api does not schedule anything, it won't need to talk to other namespaces, investigator will do its actions. I guess we can remove some service accounts and role bindings now

Reply:
ah i see, then we can remove these parts. 
other content is need for retrieving the adviser pod logs.
about the role for schedule we can remove them in separate pr.

Reply:
Sounds good to me, thanks Harshad!!